CONTRIBUTING.md

```diff
@@ -2,8 +2,15 @@ TigerOS
 ====================
 
 ## Contributing
-Please fork this repository and submit pull requests against the devel branch. Pull requests will be reviewed as time permits.
-Depending on the change you may also fork the repo and submit your PR against a new feature branch that you created. This is preferred if adding new code or items and not just modifying existing contents of the repo.
+1. Fork this repository. 
+2. Create a new branch in your fork for working on your changes.
+3. Submit a pull request with your changes against the devel branch of the RITlug/TigerOS repo.
+
+This helps prevent issues when merging your pull requests. For instructions on configuring your git environment to match with our workflow please see [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/). Pull requests will be reviewed as time permits.
+
+You can find more information about the way that GitHub interacts with git [here](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
```

Comment:
Since the section above AND below are bullets/lists, at this point this needs to be as well.